### PR TITLE
various changes and comments

### PR DESCRIPTION
**Pull request by hardaker**
Here are various edits for various purposes (clarifications, additions, etc)

### CODE REVIEW

File: draft-dns-zone-digest.xml [hunk 1]
```diff
@@ -429,12 +428,14 @@ example.com. 86400 IN &RRNAME; ( 2018031500 4 FEBE3D4CE2EC2FFA4BA9
         <t>
           Following addition of the placeholder record, the zone MAY be signed with DNSSEC.
           Note that when the digest calculation is complete, and the &RRNAME; record is updated,
-          the signature(s) for that record MUST be recalculated and updated as well.
+          the signature(s) for that record MUST be recalculated and
+          updated as well.  (Alternatively, the calculation of that
+          RRSIG value can be delayed until after the steps below are completed).
```
Comment (wessels) [inline draft-dns-zone-digest.xml:433]: I feel like the new parenthetical remark is redundant?  The start of the sentence says "when the digest calculation is complete" and the new text says "steps below are completed".  How are these different?

File: draft-dns-zone-digest.xml [hunk 2]
```diff
@@ -515,14 +516,20 @@ RR(i) = owner | type | class | TTL | RDATA length | RDATA
             The zone digest is calculated using the algorithm
             described in <xref target="calculating-calculating"/>.
             Note in particular that the Digest field of any &RRNAME;
-            RRs MUST be zeroed prior to calculation of the digest.
+            RRs MUST be zeroed prior to calculation of the digest and
+            the RRSIGs for it must be removed.
```
Comment (wessels) [inline draft-dns-zone-digest.xml:520]: I avoiding saying that the RRSIG should be removed.  Instead I said it should be excluded from the digest calculation (section "Inclusion/Exclusion Rules").  My reasoning is that we want the DNSSEC signer to have all the right things in place, including NSEC, just before the digest is calculated.  Removing and then adding back the RRSIG seems more complex to me than just updating its RDATA.

File: draft-dns-zone-digest.xml [hunk 3]
```diff
@@ -515,14 +516,20 @@ RR(i) = owner | type | class | TTL | RDATA length | RDATA
...
             The calculated digest is compared to the received digest.
             If the two digest values match, verification is considered
             successful.  Otherwise, verification MUST NOT be
             considered successful.
           </t>
+          <t>
+	    If the zone is to be served and transferred, the original
+	    Digest value of the &RRNAME; must be replace the zeroed field so
+	    downstream clients can retrieve and verify it also.
+          </t>
```
Comment (wessels) [inline draft-dns-zone-digest.xml:532]: Yeah, I see what you're getting at here.  I think this point really belongs in the (6th) item where it talks about zeroing the digest field.  That item could be worded better.

File: draft-dns-zone-digest.xml [hunk 4]
```diff
@@ -593,7 +600,13 @@ RR(i) = owner | type | class | TTL | RDATA length | RDATA
...
           We could allow a name server implementation to respond to
           &RRNAME; queries with the REFUSED RCODE without loss of
-          functionality.
+          functionality.  Note that refusal would prevent ensuring that a
+          zone-walk is complete. 
+        </t>
+        <t>
+	  General record modification, addition or deletion will
+	  be caught by checking either the value of the &RRNAME;
+	  or regular DNSSEC validation.
```
Comment (wessels) [inline draft-dns-zone-digest.xml:609]: This new last paragraph seems out of place in the "Attacks Utilizing the Zone Digest" section..?

File: draft-dns-zone-digest.xml (hunk 2 above)
Comment (hardaker) [reply draft-dns-zone-digest.xml:520]: That makes sense.  I just wanted to specifically mention that ZONEMD RRSIGs aren't included in the hash.

**Pull request merged**